packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -28,9 +28,16 @@ export interface IPersonaSharedProps extends React.HTMLAttributes<PersonaBase |
 
   /**
    * Optional custom renderer for the coin
+   *Will be @deprecated
```

Comment:
Instead of these two lines just do one line: 
`* @deprecated Use `onRenderPersonaCoin` for custom rendering instead`

Reply:
Actually `onRenderCoin` seems to have a totally different purpose (rendering the coin itself) than `onRenderPersonaCoin` (rendering the coin plus presence and stuff), so I'm not sure it should be deprecated.

Reply:
You've ran into exactly the issue why onRenderCoin should be deprecated. It doesn't actually replace the coin :( `onRenderCoin` === `onRenderImage`

It replaces only the image. When there is no image there is no way to replace the coin. Just the rendering of the text initials.

I think we can consider the presence part of the coin (as the presence is rendered by the coin). Ideally we'd solve all this stuff by introducing slots but that'll need to wait till the next version :)

Reply:
Changed the deprecated to one line